client/core/core.go

```diff
@@ -494,10 +495,13 @@ func (dc *dexConnection) compareServerMatches(srvMatches map[order.OrderID]*serv
 				continue
 			}
 			if mt.Match.Status != order.MatchStatus(msgMatch.Status) {
-				statusConflicts = append(statusConflicts, &matchStatusConflict{
-					trade: match.tracker,
-					match: mt,
-				})
+				oid := match.tracker.ID()
```

Comment:
I'd suggest to get `oid` from `for oid, match := range srvMatches` to make it that much clearer that each msgMatch is for the same trade in the inner loop.